src/functionHandlers/notarisePdt/handler.ts

```diff
@@ -62,7 +62,7 @@ export const notarisePdt = async (
       if (!signedEuHealthCerts.length) {
         throw new Error("Invalid EU vacc cert generated");
       }
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
hmm i think can omit the typing for error

Reply:
After update, the try catch start throw error `TS1196: Catch clause variable type annotation must be ‘any’ or ‘unknown’ if specified.` in linting. that why i set it as `any`, it can be any kind of throw error. [https://typescript.tv/best-practices/error-ts1196-catch-clause-variable-type-annotation/]